<?xml version="1.0" encoding="UTF-8"?><testrun name="CustomSentimentPipelineTest" project="sentiment" tests="3" started="3" failures="1" errors="0" ignored="0">
  <testsuite name="com.asynchrony.nlp.sentiment.CustomSentimentPipelineTest" time="3.129">
    <testcase name="testSentimentPipelineMainCode_Neutral" classname="com.asynchrony.nlp.sentiment.CustomSentimentPipelineTest" time="2.851"/>
    <testcase name="testSentimentPipelineMainCode_Positive" classname="com.asynchrony.nlp.sentiment.CustomSentimentPipelineTest" time="0.101">
      <failure>
        <expected>Positive</expected>
        <actual>Very negative</actual>org.junit.ComparisonFailure: Test Case [0] = Dave inturrupted the conversation and wasted everyone's time. expected:&lt;[Posi]tive&gt; but was:&lt;[Very nega]tive&gt;
	at org.junit.Assert.assertEquals(Assert.java:115)
	at com.asynchrony.nlp.sentiment.CustomSentimentPipelineTest.testParticularSentiment(CustomSentimentPipelineTest.java:56)
	at com.asynchrony.nlp.sentiment.CustomSentimentPipelineTest.testSentimentPipelineMainCode_Positive(CustomSentimentPipelineTest.java:38)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)

</failure>
    </testcase>
    <testcase name="testSentimentPipelineMainCode_Negative" classname="com.asynchrony.nlp.sentiment.CustomSentimentPipelineTest" time="0.177"/>
  </testsuite>
</testrun>
